index.html

```diff
@@ -1,7 +1,53 @@
 <!DOCTYPE html>
+<html class="no-js" lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/foundation.min.css">
+    <link rel="stylesheet" href="css/app.css">
+    <!-- Remove next link in final -->
+    <link rel="stylesheet" href="css/testing.css">
+    <!-- Below Calls to Google Fonts API -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&family=Questrial&display=swap" rel="stylesheet">
```

Comment:
Should we move some/all of these to the footer?

Reply:
Those that we keep should be moved there, yes.